src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -114,80 +115,56 @@ function QuickTracePills({
 
   if (root) {
     nodes.push(
-      <EventNodeDropdown
+      <EventNodeSelector
         key="root-node"
-        organization={organization}
         location={location}
+        organization={organization}
         events={[root]}
-      >
-        <Tooltip
-          position="top"
-          containerDisplayMode="inline-flex"
-          title={t('View the root transaction in this trace.')}
-        >
-          <EventNode type="white" pad="right" icon={null}>
-            {t('Root')}
-          </EventNode>
-        </Tooltip>
-      </EventNodeDropdown>
+        text={t('Root')}
+        hoverText={t('View the root transaction in this trace.')}
```

Comment:
Hmm thoughts on doing `{transaction_name}: {duration}` in the tooltip instead? More informative than just restating what's in the node already

Reply:
We could, though the tooltips can get rather long there if we don't truncate it. Might be good to revisit when we clean up the truncation for the dropdown.

Reply:
I'm gonna fix up truncation now so i'll update these tooltips as part of that change :+1: